example/main.go

```diff
@@ -2,14 +2,31 @@ package main
 
 import (
 	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
 
-	"github.com/getlantern/pacon"
+	"github.com/getlantern/pac"
 )
 
 func main() {
-	pacon.PacOn("localhost:12345/pac")
-	fmt.Println("proxy set, any key to continue...")
+	pac.SetHelperPath(absPath("./pac-cmd"))
+	if runtime.GOOS == "darwin" {
+		pac.SetIconPathOnOSX(absPath("icon.png"))
```

Comment:
Storing the icon as global state in the pac package doesn't feel right to me.  Better would be to just make the icon and prompt parameters to pac.On().  That way, multiple goroutines can use this library without problems.